src/controllers/productController.js

```diff
@@ -163,8 +140,6 @@ class productController{
 
   static async getAllProducts(req, res) {
     try{
-     
-   console.log(req.user);
    if (req.user.roleName !== "vendor") {
             return res.status(401).json({
```

Comment:
- responses need to be consistent 

`return res.status(401).json({
              status: "fail", message "unauthorized"});`